src/main/java/com/photolocos/enterprise/dao/ILocationDAO.java

```diff
@@ -44,7 +44,7 @@
      * @param city  the City that the user wants to get more info about photography locations
      * @return the location object found in specific state and city
      */
-    LocationDTO fetchByArea(String state, String city);
+    LocationDTO fetchByStateAndCity(String state, String city) throws Exception;
 
     /**
      * This method will return Location Objects {@link LocationDTO} based on a description
```

Comment:
DAO layer has the potential to throw Exceptions. Also made longitude and latitude a double instead of a String. Consider making them BigDecimal if greater precision is needed.